pkg/tenancy/testdata/example.yaml.golden

```diff
@@ -0,0 +1,117 @@
+apiVersion: v1
```

Comment:
.golden? ✨

Reply:
That's it exactly.

This reduces the ongoing cost as we generate more resources...we don't care about the contents as much as we care that they are being generated...

We didn't go all-in https://speakerdeck.com/mitchellh/advanced-testing-with-go?slide=19 but this is a start on this.